scene/3d/vehicle_body_3d.h

```diff
@@ -50,7 +50,7 @@ class VehicleWheel3D : public Node3D {
 	Vector3 m_wheelAxleCS; // const or modified by steering
 
 	real_t m_suspensionRestLength = 0.15;
-	real_t m_maxSuspensionTravelCm = 500.0;
+	real_t m_maxSuspensionTravelCm = 20.0;
```

Comment:
Unrelated to this PR, but it's really weird that this isn't in meters like every other value.